gadget/install/encrypt.go

```diff
@@ -67,7 +67,7 @@ func newEncryptedDeviceLUKS(part *gadget.OnDiskStructure, key keys.EncryptionKey
 		node: fmt.Sprintf("/dev/mapper/%s", name),
 	}
 
-	if err := secbootFormatEncryptedDevice(key, name+"-enc", part.Node); err != nil {
+	if err := secbootFormatEncryptedDevice(key, name, part.Node); err != nil {
```

Comment:
I'm not sure this change is a win immediately. We need to decide what is LaidOutStructure.Label and whether it needs  to be split in different fields/methods.